File: Fase4.java

public class Fase4 {

	public static void main(String[] args) {
		
		String nom, cognom1, cognom2, nom_complet;
		nom="Juan";
		cognom1=" Perez";
		cognom2=" de Asis";
		nom_complet = nom + cognom1 + cognom2;
		
		
		String dia, mes, any, data_naixement;
		dia="25";
		mes="12";
		any="1978";
		data_naixement = dia + "/" + mes + "/" + any;
		
		int any_naixement = Integer.parseInt(any); 
		
		boolean any_traspas = false;
		
		if ((any_naixement % 4 == 0 && any_naixement % 100 != 0) || (any_naixement % 100 == 0 && any_naixement % 400 == 0)) {
	           
			any_traspas = true;
		}
	
		System.out.println ("El meu nome es " + nom_complet);
		System.out.println ("Vaig néixer el " +  data_naixement);
				
		
		if (any_traspas) {
			System.out.println ("El meu any de naixement és de traspàs.");  
		          
       	}
		else {
			System.out.println("El meu any de naixement no és de traspàs.");
		}
		
	}

}

File: Fase4.js

function funcionFase4() {
		var  nom = "Juan", cognom1 = " Perez", cognom2 = " de Asis",				
			 dia = 25, mes = 12, any = 1978,
			 nom_complet = nom + cognom1 + cognom2,
			 data_naixement = dia + "/" + mes + "/" + any,
						
			 any_traspas = ((any % 4 == 0 && any % 100 != 0) || (any % 100 == 0 && any % 400 == 0));
		
		document.write ("El meu nome es " + nom_complet);
		document.write("<br>");
		document.write ("Vaig neixer el " +  data_naixement);
		document.write("<br>");
						
		
		if (any_traspas) {
			document.write("El meu any de naixement(" + any + ") es de traspas.");  
		          
       	}
		else {
			document.write("El meu any de naixement(" + any + ") no es de traspas.");
		}
	 	setInterval("location.reload()",4000);
}
